### ISSUE TITLE
Does not work in macOS 10.15 Catalina with “Auto” selected

### ISSUE THREAD

**Issue by smockle** [LinusU/DarkModeListener#1]
macOS 10.15 Catalina adds a new “Auto” option under “Appearance”:
<img width="357" alt="appearance-auto" src="https://user-images.githubusercontent.com/3104489/59219927-99e5be00-8b91-11e9-9b2a-7003e2af2bb9.png">

`DarkModeListener` logs `dark` and `light` when those options are selected, as expected. When “Auto” is selected, nothing is logged.

Related:
```
$ defaults read -g AppleInterfaceStyle
2019-06-10 15:04:46.292 defaults[1881:62752] 
The domain/default pair of (kCFPreferencesAnyApplication, AppleInterfaceStyle) does not exist
```

**Comment by ruiaureliano** [comment]
After some digging, I finally, figure this out  :)

- https://medium.com/@ruiaureliano/check-light-dark-appearance-for-macos-mojave-catalina-fb2343af875f
- https://github.com/ruiaureliano/macOS-Appearance

**Comment by cprecioso** [comment]
@ruiaureliano I don't think that method will solve the issue, as it tells you which preference is selected, but not which appearance is currently under effect.

The app needs to observe not a `UserDefaults` change, but a change in either `NSAppearance.current` or `NSApplication.shared.effectiveAppearance`; or listen to a notification the `DistributedNotificationCenter` with the name `AppleInterfaceThemeChangedNotification`.

Unfortunately, `NSAppearance.current` is not observable through KVO, and the other two approaches are not useful as macOS only delivers them to applications that show UI (not to command-line utilities - tested on Catalina).

@LinusU Since this problem has not been resolved, I would suggest for the issue to be reopened, although in the current state of affairs, I don't see a way of improving the situation save of showing some UI (e.g. a menubar item).

**Comment by LinusU** [comment]
Hmmm, this is unfortunate indeed 🤔 

Would love for any ideas on how to solve this, preferably without showing UI (which maybe isn't possible though?)

Hopefully I'll get some time to dig in to this soon
